aoc/src/main/scala/aoc2015/Day01_2015.scala

```diff
@@ -0,0 +1,39 @@
+object Day01_2015 extends App:
```

Comment:
common scala and java coding styles use `UpperCamelCase` naming for types and `lowerCamelCase` naming for values, underscores, though allowed are not used.

additionally, scala files in subdirectories mirror their content in packages, ie.

```
package aoc2015

object Day01 extends App:
  ...
```